src/jdk.httpserver/share/classes/com/sun/net/httpserver/Authenticator.java

```diff
@@ -71,11 +89,19 @@ public int getResponseCode() {
     public static class Success extends Result {
         private HttpPrincipal principal;
 
+        /**
+         * Creates a {@code Success} instance with given {@code Principal}.
+         *
+         * @param p The authenticated user you wish to set as Principal
+         */
         public Success (HttpPrincipal p) {
             principal = p;
         }
         /**
          * returns the authenticated user Principal
```

Comment:
Capitalize and add a period.
Check all the method for the first line comments.

Reply:
Will change in https://bugs.openjdk.java.net/browse/JDK-8252822